rest/snippets/UserRecordings/RecordingGroupEntity.yml

```
@@ -39,3 +50,18 @@ RecordingGroupEntity:
               type: string
               description: The ID of the series resource
               example: "abdf8785-cb23-445b-946f-69e8770500a1"
+        channel:
+          type: object
+          nullable: true
+          required:
+            - type
+            - id
+          properties:
+            type:
+              type: string
+              description: Type of the related resource, must be "Channel"
+              example: "Channel"
+            id:
+              type: string
+              description: The ID of the channel resource
+              example: "abdf8785-cb23-445b-946f-69e8770500a1"
```

Let's use a consistent ID for a channel resource in every example. The Channel entity has "42" as the example value for ID.
```suggestion
              example: "42"
```